Doc/reference/datamodel.rst

```diff
@@ -1229,17 +1230,18 @@ A class can implement certain operations that are invoked by special syntax
 (such as arithmetic operations or subscripting and slicing) by defining methods
 with special names. This is Python's approach to :dfn:`operator overloading`,
 allowing classes to define their own behavior with respect to language
-operators.  For instance, if a class defines a method named :meth:`__getitem__`,
-and ``x`` is an instance of this class, then ``x[i]`` is roughly equivalent
-to ``type(x).__getitem__(x, i)``.  Except where mentioned, attempts to execute an
-operation raise an exception when no appropriate method is defined (typically
-:exc:`AttributeError` or :exc:`TypeError`).
+operators.  For instance, if a class defines a method named
+:meth:`~object.__getitem__`, and ``x`` is an instance of this class, then
```

Comment:
A general note: it’s fine to make a changed line a bit too long or quite short to avoid having to rewrap untouched lines.  This preserves useful history for the other lines!

Reply:
Thanks! Should I rework the PR to address that, or is that more of a "thing to remember for next time" kind of thing? 🙂

Reply:
If you don’t mind the busywork, it would be ideal to minimize changes in this PR.
But I don’t want to burden volunteers!

Reply:
Done 👍